sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -301,9 +298,6 @@ case class AlterTableSerDePropertiesCommand(
     "ALTER TABLE attempted to set neither serde class name nor serde properties")
 
   override def run(sparkSession: SparkSession): Seq[Row] = {
-    DDLUtils.verifyTableProperties(
```

Comment:
users can set any data source options, but AFAIK, we only handle `path` specially, I still don't understand why we need this limitation.